client/src/components/chess-board/ChessBoard.js

```diff
@@ -1,11 +1,95 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import './ChessBoard.css';
 import ChessSquare from '../chess-square/ChessSquare';
 
-const ChessBoard = () => (
-  <div>
-    ChessBoard Component
-    <ChessSquare />
-  </div>
-);
+const matchingStringInArray = array => str => array.some(x => x.includes(str));
+
+class ChessBoard extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      possibleMoves: [],
+      from: '',
+      prevSelectValidTouch: false,
+    };
+
+    this.handleSquareClick = this.handleSquareClick.bind(this);
```

Comment:
You dont have to use this if you are using arrow functions in react component.

Reply:
Done